checkbox-provider-kivu/bin/gpu-load.py

```diff
@@ -42,8 +43,48 @@ def load_intel(fname : str, timeout : int) -> float:
 
     return compute_avg_intel(fname)
 
+def get_amd_vcn():
```

Comment:
this returns one of three arbitrary int numbers, not a VCN.
Looking at the code below it looks like the function checks if VCN is enabled so I guess `is_vcn_enabled` is much a better name. It implies a boolean as a return value and an exception on error.

Reply:
So it's always 0 or 100%? So it's still very binary to me.

Reply:
yes that is 0% or 100%, because VCN info we can get is boolean so i mapped it into 0% and 100%, i want to keep this unit (load as usage percentage) to be similar to the unit of the value we extract from radeontop and intel_gpu_top, all these two output a usage percentage ...

Reply:
and about the binary aspect of the value, yes, in real, that is binary but theoretically, it is a floating number representing a usage percentage of the video core ...